README.md

```diff
@@ -61,11 +61,12 @@ In `Java` files, you will benefit with:
   * Validation and Quick Fix support for MicroProfile
   * Code Lens support for Quarkus/MicroProfile
   * Code snippets:
-    * **qrc** - Create a new Quarkus resource class
-    * **qrm** - Create a new Quarkus resource method
     * **qtrc** - Create a new Quarkus test resource class
-    * **qntrc** - Create a new Quarkus native test resource class
-    * snippets for MicroProfile annotations.
+    * **qitrc** - Create a new Quarkus integration test resource class
+    * **qntrc** - Create a new Quarkus native test resource class (only works in older versions of Quarkus, since this testing method has been deprecated)
+    * snippets for MicroProfile annotations, coming from [vscode-microprofile](https://github.com/redhat-developer/vscode-microprofile), such as:
+      * **jaxrc** - Create a new JAX-RS resource class
```

Comment:
I think we should avoid maintaining lsp4mp snippet here. I suggest that youmove those 2 snippets in vscode-microprofile and in this README you references those snippets with the url https://github.com/redhat-developer/vscode-microprofile#microprofile-java-features

Reply:
Okay, I've made a PR to add them to the README for vscode-microprofile (see https://github.com/redhat-developer/vscode-microprofile/pull/125), and added a link in the vscode-quarkus README